### ISSUE TITLE
Feature request: loading prop for Select, Multi-Select component 

### ISSUE THREAD

**Issue by simkimsia** [palantir/blueprint#3253]
#### Environment

- __Package version(s)__: at least for v3

#### Feature request

A loading prop to components like Select as per mentioned in this comment https://github.com/palantir/blueprint/issues/2528#issuecomment-418816808

**Comment by giladgray** [comment]
yep thanks for filing this request. will slate for Q1 2019.

**Comment by simkimsia** [comment]
Welcome @giladgray  😄 Selfishly, I also need this.

**Comment by nickensoul** [comment]
Nice to see this request.
Any news on this? 😄

**Comment by jondavidjohn** [comment]
Has this been implemented?

**Comment by adidahiya** [comment]
Nope, I will close the issue when it's implemented. Until then you can vote for issues by reacting to the original post.

**Comment by phungtuanhoang1996** [comment]
^ Hi all I have a WIP PR on this feature. Currently have the omnibar working, will finish with other components later.
Also I used a `Spinner` as the default loading component. Any suggestion for a better looking option is welcome. Tks!

![ezgif com-gif-maker](https://user-images.githubusercontent.com/24819126/102099161-f69af680-3e62-11eb-8e70-0713405f1f0b.gif)
